src/main/java/mate/academy/onlinebookstore/controller/AuthenticationController.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.onlinebookstore.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import mate.academy.onlinebookstore.dto.user.UserRegistrationRequestDto;
+import mate.academy.onlinebookstore.dto.user.UserResponseDto;
+import mate.academy.onlinebookstore.exception.RegistrationException;
+import mate.academy.onlinebookstore.service.user.UserService;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "User management", description = "Endpoints of management users.")
+@RequestMapping(value = "/api/auth", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
add `server.servlet.context-path=/api `to application.properties
```suggestion
@RequestMapping("/auth")
```